iib/web/models.py

```diff
@@ -815,7 +815,10 @@ def validate_graph_mode(graph_update_mode: Optional[str], index_image: Optional[
                 f'"graph_update_mode" must be set to one of these: {graph_mode_options}'
             )
         allowed_from_indexes: List[str] = current_app.config['IIB_GRAPH_MODE_INDEX_ALLOW_LIST']
```

Comment:
Is there a plan to update IIB_GRAPH_MODE_INDEX_ALLOW_LIST config? 
lets say we want to use graph_update_mode on registry.stage.redhat.io/redhat/community-operator-index:v4.17, with this approach IIB will raise an error for 4.17 index, I will still have to make sure that the 4.17 index is added to the config list, which we are trying to avoid

Reply:
Please check now. Old rows from the config should work as well as the potential new ones (without tags).